pkg/scheduler/scheduler.go

```diff
@@ -481,9 +467,8 @@ func (sched *Scheduler) scheduleOne(ctx context.Context) {
 		// into the resources that were preempted, but this is harmless.
 		nominatedNode := ""
 		if fitError, ok := err.(*core.FitError); ok {
-			if sched.DisablePreemption || !prof.HasPostFilterPlugins() {
-				klog.V(3).Infof("Pod priority feature is not enabled or preemption is disabled by scheduler configuration." +
-					" No preemption is performed.")
+			if !prof.HasPostFilterPlugins() {
+				klog.V(3).Infof("No PostFilter plugins are registered, so no preemption will be performed.")
```

Comment:
This is preexisting, and maybe I'm misunderstanding, but this makes it seem like you have to enable postfilter plugins for preemption to work. That's pretty surprising to me.

Reply:
> this makes it seem like you have to enable postfilter plugins for preemption to work.

True, it's enabled by default. The background is that scheduler have almost "pluginize" all hard-coded logic into plugins. In 1.19 the defaultpreemption plugin follows that direction.

So for a default scheduler configuration, preemption functions the same as before. But users can still disable preemption by tweaking the config, just like what they did by specifying `DisablePreemption: false`.